### PR TITLE
fix: fixed user profile and propTypes

### PR DESCRIPTION
**Pull request by Stanislav-Vyhnan**
dev
## GitHub Project

* [Main GitHub Project ticket](https://github.com/ita-social-projects/EventsExpress-Frontend/projects/1)

## Code reviewers

- [ ] All

### Second Level Review

- [ ] @andrii-kuzmichov | @VadymKharyton 

## Summary of issue

Profile incorrect worked and some errors were in user profile

## Summary of change

Changed some logic in user-profile and fixed errors with propTypes and key in some files

## CHECK LIST
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/components/profile/user-profile.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-﻿import React, { Component } from "react";
+﻿/* eslint-disable sonarjs/no-identical-functions */
```
Comment (VadymKharyton) [inline src/components/profile/user-profile.js]: Does this eslint-disable need here?

File: src/components/profile/user-profile.js [hunk 2]
```diff
@@ -103,11 +105,9 @@ class UserItemView extends Component {
                     TransitionComponent={Zoom}
                   >
                     <IconButton
-                      className={attitude === "0" ? "text-success" : ""}
+                      className={attitude === 0 ? "text-success" : ""}
```
Comment (VadymKharyton) [inline src/components/profile/user-profile.js]: `className={!attitude ? "text-success"}`

File: src/components/profile/user-profile.js [hunk 3]
```diff
@@ -103,11 +105,9 @@ class UserItemView extends Component {
...
                       onClick={
-                        attitude !== "0"
-                          ? this.props.onLike
-                          : this.props.onReset
+                        attitude !== 0 ? this.props.onLike : this.props.onReset
```
Comment (VadymKharyton) [inline src/components/profile/user-profile.js]: As well, simplify to `attitube ? ... : ...`

File: src/components/profile/user-profile.js [hunk 4]
```diff
@@ -166,7 +166,9 @@ class UserItemView extends Component {
               <Tab
                 label="Future events"
                 icon={
-                  <IconButton color={this.state.value === 0 ? "" : "primary"}>
+                  <IconButton
+                    color={this.state.value === 0 ? "default" : "primary"}
```
Comment (VadymKharyton) [inline src/components/profile/user-profile.js:169]: As well simplify this condition

File: src/containers/local-spinner.js [hunk 5]
```diff
@@ -8,13 +9,13 @@ function LocalSpinnerWrapper({ localCounter, children, showContent }) {
...
-  showContent: PropTypes.func,
-  children: PropTypes.object,
+  showContent: PropTypes.bool,
+  children: PropTypes.array,
   localCounter: PropTypes.number,
 };
 LocalSpinnerWrapper.defaultProps = {
-  showContent: () => {},
-  children: {},
+  showContent: null,
```
Comment (VadymKharyton) [inline src/containers/local-spinner.js]: showContent specifies in `propTypes` as bool. So it should have default value as `false`

File: src/components/profile/user-profile.js (hunk 2 above)
Comment (Depesi) [reply src/components/profile/user-profile.js]: Change only in first example, if change in another examinations like and dislike don`t work correct

File: src/components/profile/user-profile.js (hunk 4 above)
Comment (Depesi) [reply src/components/profile/user-profile.js:169]: if I change same examinations in file functional of like and dislike don`t work correct

Comment (VadymKharyton) [reply src/components/profile/user-profile.js:169]: Ok, thanks for verification

**Review by VadymKharyton — APPROVED** [review]

**Pull request merged**